images/runtime/node/template.Dockerfile

```diff
@@ -12,15 +12,15 @@ ENV RELEASE_TAG_NAME=${RELEASE_TAG_NAME}
 ENV GIT_COMMIT=${GIT_COMMIT}
 ENV BUILD_NUMBER=${BUILD_NUMBER}
 #Bake in client certificate path into image to avoid downloading it
```

Comment:
Could you please remove this comment? #Bake in client certificate path into image to avoid downloading it

Reply:
Done. Any reason why it may be good to remove it?

Reply:
Oh sorry I'm now looking at the code change: I didn't mean to remove the comment everywhere. Just the one that wasn't referring to anything.

Reply:
Got it. Thanks :)